content/blogposts/2022/sourcegraph-4.0.md

```diff
@@ -0,0 +1,247 @@
+---
+title: "Sourcegraph 4.0"
+publishDate: 2022-09-27T10:00-07:00
+description: Announcing Sourcegraph 4.0
+tags: [blog, release]
+slug: "release/4.0"
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/4.0/release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/4.0/release.png
+changelogItems:
+---
+
+WIP INTRO: Sourcegraph 4.0, the latest version of our code intelligence platform, is now available. Our code intelligence platform includes a dozen new features to grok code, spend more time in flow, and execute big decisive refactors across your codebase.
```

Comment:
@lcolston08 @iskyOS - what about something like this for the intro (pulled from the CIP post):

We've spent 9 years building the underlying foundation of code search to be fast, secure, and scalable across ~10^11 lines of code for the world's most demanding software teams. Now, we’re hard at work making Sourcegraph even smarter.

Sourcegraph 4.0 includes over a dozen updates and new features, and is the first step toward Sourcegraph becoming a [code intelligence platform](/blog/code-search-to-code-intelligence). 

Read on to discover everything that shipped in Sourcegraph 4.0.

Reply:
I think it works. I personally think we need to have some type of answer to what a code intelligence platform is, but I guess with the link to the blog, we're hoping the reader gets the idea from the narrative.

Reply:
@abschu and @iskyOS What about the BLUF approach by focusing on the release first and the backstory second? 

Sourcegraph 4.0, the latest version of our code intelligence platform, is now available. Our code intelligence platform includes a dozen new features that make the platform more useful for every dev.

We've spent 9 years building the underlying foundation of code search to be fast, secure, and scalable across ~10^11 lines of code for the world's most demanding software teams. Now, we’re hard at work making Sourcegraph even smarter, is the first step toward Sourcegraph becoming a [code intelligence platform](https://github.com/blog/code-search-to-code-intelligence).

Read on to discover everything that shipped in Sourcegraph 4.0.